docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -1,128 +1,166 @@
-[[profiling-get-started]]

```

Comment:
I think they were automatically by my editor since carriage returns aren't usually in source code unless explicitly needed for Windows. I added them back to reduce the diff size for now. Aside from the profiling docs, I didn't see carriage returns in other document files.